### ISSUE TITLE
An xref target (i.e., anchor) that contains a non-ascii character causes a warning

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-pdf#283]
If an anchor contains a non-ascii character, the PDF engine shows a warning in the console. It appears that the anchor still works correctly.

Example:

```
:idprefix:
:notitle:

== Введение

Introduction text.

...

<<Введение, Введение>>
```

Here's the warning:

```
.../pdf-core-0.6.0/lib/pdf/core/pdf_object.rb:61: warning: regexp match /.../n against to UTF-8 string
```

**Comment by mojavelinux** [comment]
This problem was resolved in pdf-core 0.6.1 (the `n` flag was removed from the regexp). We may still hex encode anchors with characters outside of the ASCII range to be safe.

**Comment by mojavelinux** [comment]
Resolved by c3955c5cd7010dc9acb0b54f0e477ec2c81750c6.
